### ISSUE TITLE
No indication to press button for challenge-response

### ISSUE THREAD

**Issue by gaudenz** [kylemanna/keepassx#15]
I configured my yubikey neo to press the button for challenge-response. This works great but there is no indication from keepassx that the button should be pressed. It would be nice to have a notification.

This is not so much a problem when unlocking the database because it's obvious that the button has to be pressed, but it's annoying on save if you have autosave enabled. You have to remember to press the button when changing entries.

**Comment by kylemanna** [comment]
Yeah, it's definitely a problem I mentioned somewhere (which now escapes me).

Problem is that it requires UI/UX changes to pop-up a dialog and wait, dismiss it, cancel, etc and I never got around to it.  Pull requests welcome for clean patches that add those features. :)
